Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,51 @@
+6. There must be sufficient memory for the code.  It is unknown how much memory the code will occupy.  Comparing similar operating code with libraries and compilation show slightly over or under 150 KB (Kilobytes) in Flash/Disk/Storage.  To account for the worst case, the board will need over 150 KB.
+7. The power requirements of the board need to be reasonably under or near 240 mW.  Here, we take a voltage supply of 12 V and current of 20 mA to get 240 mW.  12 V with 20 mA is from the power subsystem’s output to provide an adequate operation time or lifespan and not damaged components.  Going over 240 mW could risk draining the battery too quickly.
+8. Some form of battery monitoring capabilities need to be included or applicable.  
+9. The board itself cannot be too big.  “Too big” here is defined as having a volume greater than 50 cubic inches.  This size was set to prevent the device from proving too large.  50 cubic inches would be about the size of a small novella/book.  .  
+10. The manufacturer needs to be a reputable and safe choice.  A stable manufacturere that is assumed to remain in business for years to come and continue producing the chosen board is a necessecity.  This project aims for replicability, so a board that remains relevant and available is a key part.
+11. The board will need to have coding language support for C and or Python.  Support for both is preferred for options and coverage but not required.  It is expected that the users will have some degree of coding experience and may potentially personalize or improve upon the code. These languages are chosen due to popularity and performance.  C, in particular, is predicted to be very desirable for this project.
+
+## BUILDABLE SCHEMATIC OF SUBSYSTEM
+Shown is the pseudo-code for the operating code.  The final form of the code may have parts added and additional functionalities included, but this is the general idea of how the code should flow.  Proceeding from each section is assuming that there are no errors detected indicating a “Success”.  If an error arises, a default handler will need to be called to assess the situation.  It should also be noted that the loop is intended to be theoretically infinite, however, there are explicit and implicit breaks at the battery running critically low (explicit) and the shutoff button being pushed (implicit).  “Timers” refer to either physical hardware or coded timers that go off to alert the system that the sensor(s) sampling is ready.
+
+![pseudo code 2](https://user-images.githubusercontent.com/110966922/202583040-c28e27fe-c175-4287-a8cb-cefafbfe327b.png)
+
+Shown is a schematic of the Arduino Mega 2560 Rev 3.
+![CAPSTONE I Shematic Checkpoint 1](https://user-images.githubusercontent.com/110966922/202655081-0386ed76-3795-4ab5-98af-ee23ff0b19f6.png)
```

Comment:
Is this a schematic drawn by the team? If so, is it helpful to have all the components of the Arduino broken out? Treat the Arduino as a single device.

Reply:
The headers shown within the schematic are all the pins from each microprocessor chip. Our Supervisor/ Customer told us to show the important connections including power and communication to each chip. Altium structures the chips in 3 sections because there are so many pins, so that is why it looks like that.